hw2/04-background.js

```diff
@@ -1 +1,41 @@
-// Add your code here
+const body = document.querySelector("#body");
+const input = document.querySelector("#input");
+const button = document.querySelector("#button");
+let intervalID = null;
+let isRunning = false;
+
+function generateRandomColorCode() {
+  const r = Math.floor(Math.random() * 256);
+  const g = Math.floor(Math.random() * 256);
+  const b = Math.floor(Math.random() * 256);
+  const colorCode = `rgb(${r}, ${g}, ${b})`;
+  return colorCode;
+}
```

Comment:
This looks great, however the assignment instructed us to use rgba or hsla and dim the colors a bit. Very similar to what you have here, but this was my solution:

function changeBackground() {
  let r = Math.floor(Math.random() * 256);
  let g = Math.floor(Math.random() * 256);
  let b = Math.floor(Math.random() * 256);
  let a = 0.8;
  let rgba = `rgba(${r}, ${g}, ${b}, ${a})`;
  document.querySelector("body").style.background = rgba;
}

Good job, code looks great!